docs/partials/_contributing.md

```diff
@@ -1 +0,0 @@
-../../CONTRIBUTING.md
```

Comment:
This file was shared between 0.2 and current docs. However since the content has now diverged, I have created it separately in both versions of the docs, with the current one modified as per @helderco feedback.

Reply:
I'm not sure I see the point in the split. The cue version is now in a separate repo so it doesn't make sense to keep CUE related instructions in this repo's `CONTRIBUTING.md`.

Reply:
Sorry, I didn't understand this comment. Which separate repo are you referring to for cue?

As additional info: currently the `0.2` and `current` docs are in the same Docusaurus instance. The relevant pages are https://docs.dagger.io/204441/contributing (`current`) and  https://docs.dagger.io/1227/contributing (`0.2`). Internally the content was shared using this partial. Since the instructions have now diverged, this commit deletes the partial and instead uses separate files for each of the above URLs.

Reply:
Sorry, I didn't mean a separate repo, I mean't it's in a separate branch ([cue-sdk](https://github.com/dagger/dagger/tree/cue-sdk)).

At this moment this PR has 3 versions of this:
- `CONTRIBUTING.md`
- `docs/current/204441-contributing.md`
- `docs/v0.2/guidelines/1227-contributing.md`

Only `current` has CUE removed, while the other two don't. It should only be removed from the `0.2` docs.

`CONTRIBUTING.md` should always be up to date, so you can keep the partial and the reference in `current`.

Reply:
So my understanding is (apologies if I have misunderstood):
- `CONTRIBUTING.md` should reflect the current situation (ie. no references to CUE)
- `docs/current/204441-contributing.md` = `CONTRIBUTING.md`
- `docs/v0.2/guidelines/1227-contributing.md` should reflect the previous situation (continue with CUE references)

Reply:
Yes, that's right 🙂

Reply:
Ok, I think the latest commit should do it @helderco